### ISSUE TITLE
Question about evaluation of PSNR.

### ISSUE THREAD

**Issue by HLinChen** [NVlabs/neuralangelo#155]
From your codes, I find you use all images for training on both tnt and dtu datasets. So what are the test images? Can you provide the indices of test images?

**Comment by hbai98** [comment]
Same, I gain a PSNR of 35.15 when validating on dtu_scan24 which surpasses the reported value 30.64 a large margin.

**Comment by mli0603** [comment]
Hi @HLinChen and @hbai98 

We do not separate the training/test images when reporting PSNR. As mentioned in the paper, we use PSNR as a metric to assess the capacity to overfit the input images and capture details. I hope this helps!

**Comment by tom-m-walker** [comment]
Hi @mli0603,

If you do not separate the training/test images for PSNR, then why are the reported PSNR scores so much lower than those obtained by the model when overfitting to all training views? I'm trying to compare to your approach, but it's unclear to me exactly how the PSNR reported in the paper were computed. 

Thanks!
